docs/source/reference/router/telemetry/metrics-exporters/datadog.mdx

```diff
@@ -8,27 +8,58 @@ Enable and configure the [OTLP exporter](/router/configuration/telemetry/exporte
 
 For general tracing configuration, refer to [Router Metrics Configuration](/router/configuration/telemetry/exporters/metrics/overview).
 
-## Datadog configuration
+## Configuration 
 
-To export metrics to Datadog, you must both: 
+To export metrics to Datadog, you must configure both the router to send traces to the Datadog agent and the Datadog agent to accept OpenTelemetry Protocol (OTLP) metrics.
 
-- Configure the Datadog agent to accept OpenTelemetry Protocol (OTLP) metrics, and
-- Configure the router to send traces to the Datadog agent.
+### Router configuration
 
-### Datadog agent configuration
+The router should set attributes that Datadog uses to organize its APM view and other UI:
 
-To configure the Datadog agent, add OTLP configuration (`otlp_config`) to your `datadog.yaml`. For example:
+- `otel.name`: span name that's fixed for Datadog
+- `resource.name`: Datadog resource name that's displayed in traces
+- `operation.name`: Datadog operation name that populates a dropdown menu in the Datadog service page
 
-```yaml title="datadog.yaml"
-otlp_config:
-  receiver:
-    protocols:
-      grpc:
-        endpoint: <dd-agent-ip>:4317
+You should add these attributes to your `router.yaml` configuration file. The example below sets these attributes for the `router`, `supergraph`, and `subgraph` stages of the router's request lifecycle:
+
+```yaml title="router.yaml"
+telemetry:
+  instrumentation:
+    spans:
```

Comment:
I think there is some confusion here. This is actually tracing config. I think this should be shifted to the datadog tracing config.

Reply:
Same goes for the new screenshot